test/Sema/exhaustive_switch.swift

```diff
@@ -1175,3 +1175,35 @@ func sr10301_as(_ foo: SR10301<String,(Int,Error)>) {
     return
   }
 }
+
+enum Untupled {
+    case upair(Int, Int)
+}
+
+func sr11212_content_untupled_pattern_tupled(u: Untupled) -> (Int, Int) {
+    switch u {
+    case .upair((let x, let y)): return (x, y)
+    }
+}
+
+func sr11212_content_untupled_pattern_tupled_nested(u: Untupled) -> (Int, Int) {
+    switch u {
+    case .upair(let (x, y)): return (x, y)
+    }
+}
+
+func sr11212_content_untupled_pattern_ambiguous(u: Untupled) -> (Int, Int) {
+    switch u {
+    case .upair(let u_): return u_
+    }
+}
+
+enum Tupled {
+    case tpair((Int, Int))
+}
+
+func sr11212_content_tupled_pattern_untupled(t: Tupled) -> (Int, Int) {
```

Comment:
Do we already have test cases for the other two?

Can you add test cases for the generic one you've been working on, even if they don't work yet?

Reply:
I don't understand which test cases you're referring to in "other two forms" and "generic one"? The table has 4 entries for code which shouldn't compile but still do (on master) -- I've added one function for each such entry. The SR-11160 fix fails two of those tests.

Reply:
You're right: there should be *12* functions here: all four forms on all three kinds of enum (Untupled, Tupled, and Generic<Tuple>).